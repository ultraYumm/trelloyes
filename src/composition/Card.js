import React from 'react';
import './Card.css';


export default function Card(props) {
  return (
    <div className='Card'>
      <button
        onClick={() => props.onClickDelete(props.id)}
        type='button'
      >
        delete
      </button>
      <h3>{props.title}</h3>
      <p>{props.content}</p>
    </div>
  )
}

Card.propTypes = {
  onClickDelete: () => {}
}