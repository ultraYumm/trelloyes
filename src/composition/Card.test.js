import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';


it('renders without crashing', () => {
    // first create a DOM element to render the component into
    const div = document.createElement('section');
    ReactDOM.render(<Card />, div);
  
    // clean up code
    ReactDOM.unmountComponentAtNode(div);
  });