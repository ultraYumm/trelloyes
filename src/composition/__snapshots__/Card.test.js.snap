// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="Card"
>
  <button
    onClick={[Function]}
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
</div>
`;
